docs/DebuggingTheCompiler.md

```diff
@@ -582,6 +583,20 @@ it's quite easy to do this manually:
      did wrong. To simplify the comparison, it's sometimes helpful to replace
      all SIL values (e.g. `%27`) with a constant string (e.g. `%x`).
 
+### Bisecting on LLVM optimizer pass counts to identify optimizer bugs
+
+One can use the tips from the previous section to also bisect the LLVM optimizer
+with slightly different options:
+
+1. `-Xllvm --opt-bisect-limit=<INT>`: Maximum number of optimizations to perform. The equivalent of `-sil-opt-pass-count`.
+2. `-Xllvm --filter-print-funcs=<function names>`: Only print IR for functions whose name match this for all print-[before|after][-all] options.
+3. `-Xllvm --print-after=<string>`: Print IR after specified passes
+4. `-Xllvm --print-after-all`: Print IR after each pass
+6. `-Xllvm --print-before=<string>`: Print IR before specified passes
+7. `-Xllvm --print-before-all`: Print IR before each pass
+8. `-Xllvm --print-before-changed`: Print before passes that change them
+
+
```

Comment:
super nit: it looks like other places in this file just use one newline before the start of a new section.